dhis-2/dhis-api/src/main/java/org/hisp/dhis/mapping/MapView.java

```diff
@@ -219,6 +220,15 @@
 
     private transient List<OrganisationUnit> organisationUnitsInGroups = new ArrayList<>();
 
+    /**
+     * The displayName of the {@link Attribute} which has ID stored by property
+     * {@link MapView#orgUnitField}
+     * <p>
+     * The value of this transient property will be set in
+     * MapController#postProcessResponseEntity
+     */
+    private transient String orgUnitFieldName;
```

Comment:
Could we call this property `orgUnitFieldDisplayName` to make the name explicit, and allow for a potential `orgUnitFieldName` with the un-translated name in the future?

Reply:
Changed to `orgUnitFieldDisplayName` now.